test/test_e2e_model.py

```diff
@@ -72,10 +75,46 @@ def init_chainer_weight_const(m, val):
             p.data[:] = val
 
 
+def test_chainer_ctc_type():
+    import logging
+    logging.basicConfig(
+        level=logging.DEBUG, format='%(asctime)s (%(module)s:%(lineno)d) %(levelname)s: %(message)s')
+    import e2e_asr_attctc as ch
+
+    out_data = "1 2 3 4"
+    numpy.random.seed(0)
+    data = [
+        ("aaa", dict(feat=numpy.random.randn(200, 40).astype(
+            numpy.float32), tokenid=out_data)),
+        ("bbb", dict(feat=numpy.random.randn(100, 40).astype(
+            numpy.float32), tokenid=out_data)),
+        ("cc", dict(feat=numpy.random.randn(100, 40).astype(
+            numpy.float32), tokenid=out_data))
+    ]
+
+    def _propagate(ctc_type):
+        args = make_arg(
+            ctc_type=ctc_type, elayers=2, eunits=20, eprojs=20
```

Comment:
if you think these args `elayers=2, eunits=20, eprojs=20` are OK , make it default in `make_arg`

Reply:
@jheymann85 @jheymann85, if this is fixed (or if @jheymann85 think that it should be as it is, I'll merge this PR.

Reply:
Ok, I kept the network configuration and adjusted the tolerance for the loss instead
(its `x: array(36.5968, dtype=float32)` vs. `y: array(36.596798, dtype=float32)`)